pyproject.toml

```diff
@@ -285,7 +285,7 @@ attr-naming-style = "snake_case"
 # Regular expression matching correct attribute names. Overrides attr-naming-
 # style. If left empty, attribute names will be checked with the set naming
 # style.
-attr-rgx = "[a-z_][a-z0-9_]{2,}$"
```

Comment:
@nfx: Need this to match the `id` attribute on `RawAzureResource`, `StorageAccount` and `AccessConnector`. Could also restrict the regex to explicitly match `id`, not any two letter attribute